connector_easypost/tests/common.py

```diff
@@ -75,13 +75,10 @@ def __init__(self, cr, registry, model_name):
         self.model = registry(model_name)
 
 
-class ObjDict(dict):
+class ObjDict(object):
 
-    def __getattr__(self, key):
-        try:
-            return super(ObjDict, self).__getattr__(key)
-        except AttributeError:
-            return self[key]
+    def __init__(self, **entries):
+        self.__dict__.update(entries)
 
```

Comment:
FWIW, this changes is because using `getattr()` on `ObjDict` objects with a missing key was raising a `KeyError`, instead of attribute error. This was killing `getattr()` calls, which would've normally not failed.